framework/src/modules/dpos_v2/module.ts

```diff
@@ -147,6 +148,20 @@ export class DPoSModule extends BaseModule {
 			throw new Error("'voteCommand' is missing from DPoS module");
 		}
 		voteCommand.init({ tokenIDDPoS: this._moduleConfig.tokenIDDPoS });
+
+		const unlockCommand = this.commands.find(command => command.id === COMMAND_ID_UNLOCK) as
+			| UnlockCommand
+			| undefined;
+		if (!unlockCommand) {
+			throw new Error("'unlockCommand' is missing from DPoS module");
```

Comment:
will be nice if `"'unlockCommand' is missing from DPoS module"` error can be tested in the unit test